src/store/modules/resetpassword/actionTypes/index.js

```diff
@@ -0,0 +1,10 @@
+const actionTypes = {
+  LOGIN_REQUEST: 'LOGIN_REQUEST',
```

Comment:
There seems to be no need for the `LOGIN_REQUEST`, `REQUEST_PENDING` and `LOGIN_SUCCESS` actionType as they are not used here. I think it would be okay to remove them

Reply:
Okay. Thanks, it will be removed as aprropriate